packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1361,18 +1361,33 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         return fluidDataStore;
     }
 
-    public createDetachedDataStore(): IFluidDataStoreContextDetached {
-        const id = uuid();
+    public createRootDetachedDataStore(
```

Comment:
Feels a bit pedantic and hopefully another native English speaker can comment as well in case I'm wrong, but...

I think this should be `createDetachedRootDataStore`?  Partly because it's a root data store that is detached (as opposed to a detached data store that is root), and partly because (I _think_) it violates the unspoken rule of adjective ordering?

Reply:
I agree :)